middleware/http/opa/opa_middleware.go

```diff
@@ -0,0 +1,178 @@
+package opa
```

Comment:
Since we are already in package `opa`, can you please rename the file to just `middleware.go`?